src/VernaForm.spec.tsx

```diff
@@ -59,26 +59,29 @@ describe('VernaForm', () => {
       </VernaContextProvider>,
     );
 
-    // Add two sections
-    screen
-      .getByRole('button', {
-        name: 'Add a section',
-      })
-      .click();
-    _.last(
-      (await screen.findAllByRole('button', {
-        name: 'Add a section',
-      })) as HTMLElement[],
-    )?.click();
+    screen.debug();
+
+    await act(async () => {
+      // Add two sections
+      screen.getByRole('button', { name: 'Add a section' }).click();
+
+      _.last(
+        (await screen.findAllByRole('button', {
+          name: 'Add a section',
+        })) as HTMLElement[],
+      )?.click();
+    });
 
     const $addFieldButtons1 = await screen.findAllByRole('button', {
       name: 'Add an input',
     });
     expect($addFieldButtons1).toHaveLength(3);
 
     // Add two input fields
-    await clickOnLastAddInputButton();
-    await clickOnLastAddInputButton();
+    await act(async () => {
+      await clickOnLastAddInputButton();
+      await clickOnLastAddInputButton();
+    });
```

Comment:
What was the need to modify the tests?
(those tests are changed in the widget pr to remove the await part and not use find so it's not async)

Reply:
Whoops a residue of tests I did to try to upgrade to React 18.